managing_providers/cloud_providers/_topics/adding_ibm_cic_providers.md

```diff
@@ -0,0 +1,63 @@
+
+The corresponding {{ site.data.product.title_short }} plugin supports IBM CIC **v1.4.4 or higher**.
+
+Complete the following procedure to add an IBM CIC cloud provider in
+{{ site.data.product.title_short }}.
+
+1.  Browse to menu: **Compute > Clouds > Providers**.
+
+2.  Click **Configuration**, then click
+    ![1862](../../images/1862.png) (**Add a New Cloud Provider**).
+
+3.  From the **Type** list, select **IBM CIC**.
```

Comment:
The type should be "IBM Cloud Infrastructure Center"

![image](https://github.com/user-attachments/assets/cb95b288-9f4b-475a-a7a9-2a80d6b4f8b4)